common/scripted_buttons/imperia_dead_man_buttons.txt

```diff
@@ -0,0 +1,172 @@
+						has_ideology = ideology:ideology_liberal_leader #democrat
+						has_ideology = ideology:ideology_positivist
+						has_ideology = ideology:ideology_market_liberal
+					}
+				}
+				is_interest_group_type = ig_intelligentsia
+			}
+		}
+		custom_tooltip = {
+			text = not_already_supporting_republicanism
+			NOT = { has_variable = supporting_republican_var }
+		}
+	}
+	
+	selected = {
+		has_variable = supporting_republican_var
+	}
+
+	effect = {
+		set_variable = supporting_republican_var
+		set_variable = has_selected_republican_once_var
+		remove_variable = supporting_theocracy_var
+		remove_variable = supporting_communist_var
+		custom_tooltip = {
+			text = republican_progress_increase_10_tt
+			change_variable = { name = republican_progress add = 10 }
+		}
+	}
+}
+
+tur_theocracy_button = {
+	name = "theocracy_button"
+	desc = "theocracy_button_desc"
+
+	visible = {
+		owner = { 
+			has_variable = chose_initial_turk_government 
+		}
+	}
+
+	possible = {
+		trigger_if = {
+			limit = {
+				has_variable = has_selected_theocracy_once_var
+			}
+			custom_tooltip = {
+				text = cannot_choose_theocracy_twice_tt
+				always = no
+			}
+		}
+		any_interest_group = {
+			is_in_government = yes
+			OR = {
+				leader = {
+					OR = {
+						has_ideology = ideology:ideology_theocrat
+						has_ideology = ideology:ideology_traditionalist
+						has_ideology = ideology:ideology_authoritarian
+						has_ideology = ideology:ideology_slaver
+						has_ideology = ideology:ideology_corporatist
+					}
+				}
+				is_interest_group_type = ig_devout
+			}
+		}
+		custom_tooltip = {
+			text = not_already_supporting_theocracy
+			NOT = { has_variable = supporting_theocracy_var }
+		}
+	}
+	
+	selected = {
+		has_variable = supporting_theocracy_var
+	}
+
+	effect = {
+		# Only one of these variables can be active at a time
+		set_variable = supporting_theocracy_var
+		set_variable = has_selected_theocracy_once_var
+		remove_variable = supporting_republican_var
+		remove_variable = supporting_communist_var
+		custom_tooltip = {
+			text = theocracy_progress_increase_10_tt
+			change_variable = { name = theocracy_progress add = 10 }
+		}
+	}
+}
+
+tur_communist_button = {
+	name = "communist_button"
+	desc = "communist_button_desc"
+
+	visible = {
+		owner = { 
+			has_variable = chose_initial_turk_government 
+		}
+	}
+
+	possible = {
+		trigger_if = {
+			limit = {
+				has_variable = has_selected_communist_once_var
+			}
+			custom_tooltip = {
+				text = cannot_choose_communist_twice_tt
+				always = no
+			}
+		}
+		any_interest_group = {
+			is_in_government = yes
+			OR = {
+				leader = {
+					OR = {
+						has_ideology = ideology:ideology_communist
+						has_ideology = ideology:ideology_vanguardist
+						has_ideology = ideology:ideology_anarchist
+						has_ideology = ideology:ideology_social_democrat
+						has_ideology = ideology:ideology_humanitarian
+					}
+				}
+				is_interest_group_type = ig:ig_trade_unions
```

Comment:
Should just be `is_interest_group_type = ig_trade_unions`
![image](https://github.com/Imperia-Development-Victoria-3/Victoria-3-Dev/assets/44327039/fdf8718b-bc9c-4002-ae5c-659d7016879b)

Reply:
But why? If any of these ideologies are leading an IG they should be able to click the button

Reply:
as opposed to ig:ig_trade_unions.

Reply:
Oh I see, thank you